src/domain/admin/layout/AdminPageLayout.tsx

```diff
@@ -18,9 +22,15 @@ const AdminPageLayout: React.FC<PropsWithChildren> = ({ children }) => {
     { label: t('keywordsPage.title'), to: ROUTES.KEYWORDS },
     { label: t('keywordSetsPage.title'), to: ROUTES.KEYWORD_SETS },
     { label: t('imagesPage.title'), to: ROUTES.IMAGES },
-    { label: t('organizationsPage.title'), to: ROUTES.ORGANIZATIONS },
-    { label: t('placesPage.title'), to: ROUTES.PLACES },
-    { label: t('priceGroupsPage.title'), to: ROUTES.PRICE_GROUPS },
+    ...(arePriceGroupRoutesAllowed(user)
```

Comment:
Although these are currently allowed for the same users, it is a tiny bit confusing when taking a first look at this and seeing that price group permissions are checked for organization routes. Creating a dedicated function for organization route permissions check or renaming this to something like `areFinancialRoutesAllowed` could make this easier to read.

Reply:
Renamed the function as `areFinancialRoutesAllowed`. Also added check for all the organization routes so user without sufficient permissions is not allowed to see the organization pages